src/api/libcellml/generator.h

```diff
@@ -69,6 +71,20 @@ class LIBCELLML_EXPORT GeneratorVariable
      */
     VariablePtr variable() const;
 
+    /**
+     * @brief Get the @c Component for this @c GeneratorVariable.
+     *
+     * Return the @c Component of this @c GeneratorVariable. It is the
+     * @c Component in which the @c Variable is first defined (in the case of
+     * the variable of integration), initialised (in the case of a constant) or
+     * actually computed (in the case of a state, computed constant or algebraic
+     * variable). It may, or not, be the same @c Component as the parent
```

Comment:
Either "It may or may not be the same ..." or "It is not necessarily the same ..."

Reply:
Thanks, fixed in 4a0fad52.